default.py

```diff
@@ -39,7 +40,10 @@ def build_url(query):
     xbmcplugin.addDirectoryItem(handle=addon_handle, url=url, listitem=li, totalItems=2)
 
     # Get a list of the games from Lutris
-    args = settings.getSetting('lutris_executable') + ' --list-games --json'
+    if settings.getSetting('custom_path') in [True, 'True', 'true', 1]:
```

Comment:
This could just be `settings.getSetting('custom_path')`. Python truthy can be confusing.